.rubocop_rspec_base.yml

```diff
@@ -1,79 +1,76 @@
-# This file was generated on 2021-07-15T10:45:53+01:00 from the rspec-dev repo.
+# This file was generated on 2021-07-15T10:45:52+01:00 from the rspec-dev repo.
```

Comment:
We generally update this file using rspec-dev that can make multiple changes at once, I'd prefer to do that and then add the extra "version bump" commit to those prs by hand.